src/commands/register.ts

```diff
@@ -331,6 +333,12 @@ export default class ComponentRegister extends BaseCommand {
       this.generateDependenciesWarnings(new_spec.dependencies, selected_account.name);
     }
 
+    for (const [service_name, service_config] of Object.entries(component_spec.services || {})) {
+      if (service_config.scaling && account_subscription.subscription_tier === 'Free') {
+        this.warn(chalk.yellow(`The service "${service_name}" contains scaling settings, but scaling is only available for the Team and Growth account tiers. The service will run normally, but scaling will not occur.`));
```

Comment:
Can we make it so the stack trace doesn't show for warning logs? See your screenshot.